hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/CapacitySchedulerConfiguration.java

```diff
@@ -1560,82 +1545,82 @@ void setWorkflowPriorityMappings(
 
   public boolean isReservable(String queue) {
     boolean isReservable =
-        getBoolean(getQueuePrefix(queue) + IS_RESERVABLE, false);
+        getBoolean(getQueuePrefix(new QueuePath(queue)) + IS_RESERVABLE, false);
     return isReservable;
   }
 
   public void setReservable(String queue, boolean isReservable) {
-    setBoolean(getQueuePrefix(queue) + IS_RESERVABLE, isReservable);
+    setBoolean(getQueuePrefix(new QueuePath(queue)) + IS_RESERVABLE, isReservable);
```

Comment:
Please only create the QueuePath object once in this method.

Reply:
Sure, fixed that too